staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset_test.go

```diff
@@ -1037,7 +1026,7 @@ func TestDifferentFlowsWithoutQueuing(t *testing.T) {
 	now := time.Now()
 
 	clk, counter := testeventclock.NewFake(now, 0, nil)
-	qsf := newTestableQueueSetFactory(clk, countingPromiseFactoryFactory(counter))
+	qsf := newTestableQueueSetFactory(clk, countingPromiseFactoryFactory(counter), test.NewFakeQueueWait())
```

Comment:
This test was using RequestWaitLimit=0

Reply:
The question is whether it's something we really want to test it - do we use it in production?